Jenkins/conda-bot.py

```diff
@@ -0,0 +1,72 @@
+import re
+import sys
+
+from packaging import version
+import conda.cli.python_api
+from github import Github
+
+
+TOKEN = sys.argv[1]
+g = Github(TOKEN)
+repo = g.get_repo('vanheeringen-lab/snakemake-workflows')
+open_pulls = list(repo.get_pulls(state='open'))
+
+# ignore
+with open('./envs/ignore.txt') as f:
+    ignore = f.read().splitlines()
+
+for file in repo.get_contents("envs", ref='develop'):
+    updated = dependencies = False
+    newfile = ''
+
+    pull_request_title = f'auto-update: {file.name}: '
+
+    lines = list(filter(None, file.decoded_content.decode("utf-8").split('\n')))
+    for line in lines:
+        # skip lines until the dependencies are listed
+        if not dependencies:
+            dependencies = 'dependencies:' in line
+        else:
+            # ignore lines starting with pip, deeper indented lines, and comments
+            if 'pip:' not in line and not line.startswith('    -') and not line.startswith('#'):
```

Comment:
Ignore a wider range of text in env.yamls
```suggestion
            # ignore lines with pip, deeper indented lines, comments and empty lines
            if '  - ' in line and 'pip:' not in line and not line.startswith(('    -', '#', '\n')):
```